activesupport/lib/active_support/core_ext/string/inflections.rb

```diff
@@ -164,7 +164,7 @@ def deconstantize
   #
   #   <%= link_to(@person.name, person_path) %>
   #   # => <a href="/person/1-donald-e-knuth">Donald E. Knuth</a>
-  def parameterize(sep = '-')
+  def parameterize(sep = '-'.freeze)
```

Comment:
Do you think it's better to have a single place that has all the frozen strings? Sort of like `boot.rb`, but that will have the calls to `freeze` for these tokens (`-` here, `::` in the next file etc.